packages/autorest.typescript/test/integration/generated/appconfiguration/package.json

```diff
@@ -47,14 +42,13 @@
     "src/**/*.ts",
     "README.md",
     "LICENSE",
-    "rollup.config.js",
     "tsconfig.json",
     "review/*",
     "CHANGELOG.md",
     "types/*"
   ],
   "scripts": {
-    "build": "npm run clean && tsc && rollup -c 2>&1 && npm run minify && mkdirp ./review && npm run extract-api",
+    "build": "npm run clean && tsc && dev-tool run bundle && npm run minify && mkdirp ./review && npm run extract-api",
```

Comment:
I am a little confused, this change is for azure-sdk-for-js: true  and here we are changing the integration test. right ? Has the integration test enabled azure-sdk-for-js: true ?

Reply:
From what I can find, this test doesn't specify azureSdkForJs option

https://github.com/Azure/autorest.typescript/blob/main/packages/autorest.typescript/test/commands/test-swagger-gen.ts#L628

By default we generate for JS repo

https://github.com/Azure/autorest.typescript/blob/c3d9f9bf3c5d45ff9fea680fe73918428e7721ed/packages/autorest.typescript/src/utils/autorestOptions.ts#L123